.github/pull_request_template.md

```diff
@@ -4,6 +4,11 @@
 
 
 ## How was the functionality tested and verified?
+All steps should be completed in the order presented, and you should not move on to the next step until the previous is completed.
+- [ ] The title of the `html` page is the same as the chart.
+- [ ] The `meta_data.json` file is filled in correctly (the description is a complete sentence, the title is the same as the visual).
+- [ ] The visual is styled similar to other charts (centered title/subtitle, no legends unless absolutely necessary, etc.)
+- [ ] This visual was approved by the lead developer.
 - [ ] The visual was approved by the story's stakeholders.
 
 <!-- Replace this comment with a screenshot of your visual. -->
```

Comment:
another random comment here is how we can better review charts that arent static (hopefully at some point). idk if this means a dev branch someone can pull locally to look at the chart or some other solution but maybe just something to noodle on

Reply:
For reviewing PRs, it's definitely encouraged that you pull down the branch and check it out. Whenever I do reviews for them, I pull the branch down, run the minification step, etc. With regard to dynamic charts, those will be done on a case-by-case basis as each chart is likely to to be quite unique in its requirements. Since this is done in JS, we won't have a scenario where secret keys are an option, so we shouldn't need a dev branch.

Reply:
makes sense!